server/util/rabbitmq/RabbitMQconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "vhosts": {
```

Comment:
Will this config file need to get edited if we're not running the queue locally?

Reply:
Replied to this in-depth on the chain-events side as well. The config file does not need to be the same for all instances (chain-events and CW) since for example the Rascal instance on CW does not need to be aware of publications. Ideally, (for the sake of simplicity and to avoid confusion) store this in the database and allow all instances (chain-events and CW) to retrieve it as making CW aware of the publications does not hurt/hinder or change the functionality of the consumer. tl;dr from chain-events: not necessarily, depends on deployment choices.